<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Avogadro::QtPlugins::ThreeDMolDialog</class>
 <widget class="QDialog" name="Avogadro::QtPlugins::ThreeDMolDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3DMol HTML Snippet</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="copyButton">
         <property name="text">
          <string>&amp;Copy to Clipboard</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
